openedx/features/course_experience/templates/course_experience/course-updates-fragment.html

```diff
@@ -36,7 +36,22 @@
         </div>
     </header>
     <div class="page-content">
-        ${HTML(updates_html)}
+        % if len(update_list_html) > 0:
+            ${HTML(update_list_html)}
+        % else:
+            <div class="all-updates">
+                % for index, update in enumerate(updates):
+                <article class="updates-article">
+                    % if not update.get("is_error"):
+                    <div class="date">${update.get("date")}</div>
+                    % endif
+                    <div class="article-content">
+                        ${HTML(update.get("content"))}
+                    </div>
+                </article>
```

Comment:
I'm guessing we need to render this separately because it doesn't let the updates be rendered in reverse chronological order? I don't think it's necessarily wrong, but I am just curious about the reasons why it was changed.